libraries/CAN/test/unit/src/features/Test_CAN_connected_node1.hpp

```diff
@@ -0,0 +1,82 @@
+// std includes
+
+// test includes
+#include "Test_includes.hpp"
+
+// project includes
+
+
+void CAN_connected_node1_suiteSetup(void);
+void CAN_connected_node1_suiteTearDown(void);
+
+
+// variables used in the tests below that have to be accessed in the setup and tear down methods
+
+
+// test includes that may require dut
+
+
+// define test group name
+TEST_GROUP(CAN_connected_node1);
+TEST_GROUP(CAN_connected_node1Internal);
+
+
+// Setup method called by Unity before every individual test defined for this test group.
+static TEST_SETUP(CAN_connected_node1Internal)
+{
+}
+
+// Tear down method called by Unity after every individual test defined for this test group.
+static TEST_TEAR_DOWN(CAN_connected_node1Internal)
+{
+}
+
+void testSendReceive() 
+{
+
+    CAN.beginPacket(CAN_ID_1);
```

Comment:
The CAN function return value is still not checked. More of these below.

Reply:
fixed